poi/src/main/java/org/apache/poi/util/UserNameAwareDefaultTempFileCreationStrategy.java

```diff
@@ -0,0 +1,57 @@
+/* ====================================================================
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+==================================================================== */
+
+package org.apache.poi.util;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.apache.poi.util.TempFile.JAVA_IO_TMPDIR;
+
+/**
+ * Username-aware subclass of {@link DefaultTempFileCreationStrategy}
+ * that avoids permission issues when deploying applications with multiple users on the same server.
+ * Other than adding the username to the temporary directory, all other behavior is the same as the superclass.
+ *
+ * @author Tiger Zhao
+ * @since 2024-10-05 22:41

```

Comment:
Thanks but can you remove this author tag. We don't use tags like this in our shared code.

`@since POI 5.3.1` too - since is for version numbers not times.